src/Components/CustomAlert.js

```diff
@@ -1,25 +1,60 @@
 import React from "react"
-import { Stack, Alert } from "@mui/material"
+import { Alert } from "@mui/material"
+import { Slide } from "@mui/material"
 
 let StyledAlert = (props) => {
-	console.log("styledAlert props", props)
-	return (
-		<Stack spacing={2}>
-			<Alert
-				variant="filled"
-				severity="error"
-				sx={{ display: `${props.errorVisible}` }}
-			>
-				{props.message}
-			</Alert>
-		</Stack>
-	)
+	let variableForSlideInProp = props.errorVisible !== "none"
+	let errorDirection = "up"
+	if (props.errorVisible === "userCreated") {
+		//returns succuss alert of user being created
+		console.log("alert props", props)
+		return (
+			<>
+				<Slide
+					direction={`${errorDirection}`}
+					in={variableForSlideInProp}
+					mountOnEnter
+					unmountOnExit
+				>
+					<Alert
+						variant="filled"
+						severity="success"
+						sx={{ display: `${props.errorVisible}` }}
+					>
+						{`User successfully created! Please log in with email '${props.userDataObj.email}' and password '${props.userDataObj.password}'`}
+					</Alert>
+				</Slide>
+			</>
+		)
+	} else {
+		//returns error alert of user not being created
+		return (
+			<>
+				<Slide
+					direction={`${errorDirection}`}
+					in={variableForSlideInProp}
+					mountOnEnter
+					unmountOnExit
+				>
+					<Alert
+						variant="filled"
+						severity="error"
+						sx={{ display: `${props.errorVisible}` }}
```

Comment:
seems like props.errorVisible is a string? You probably want it to be just a boolean then `${props.errorVisible ? 'block' : 'none'}` or whatever values for display you are shooting for.

Reply:
so i have props.errorVisible as a string specifically because of the sx proprerty use from mui. 'none' causes the error window not to be shown, (not even rendered if I remember correctly?) I originally had the errorVisable state set to a boolean, but switched to 'none' and other strings. Would you have approached differently?